src/store/proposals/index.js

```diff
@@ -419,7 +419,14 @@ export default {
             { label: 'ballot_description', value: ['string', draft.description] }
           ]
         } else {
-          if (draft.badge.details_title_s === 'Voter' || draft.badge.details_title_s === 'Delegate') {
+          if (
+            (draft &&
+              draft.badge &&
+              draft.badge.details_title_s === 'Voter') ||
```

Comment:
@arsenijesavic `Voter` and `Delegate` should be constants, not strings. Preferably shared between all components that need those.